zerver/models/streams.py

```diff
@@ -186,6 +186,9 @@ def is_history_realm_public(self) -> bool:
     def is_history_public_to_subscribers(self) -> bool:
         return self.history_public_to_subscribers
 
+    def is_support_stream(self) -> bool:
```

Comment:
I guess this commit will fail coverage because this fucntion is not used anywhere as of this function.

Reply:
Fixed.